modules/porous_flow/doc/content/modules/porous_flow/natural_convection.md

```diff
@@ -0,0 +1,94 @@
+# Natural Convection
+
+## Problem statement
+
+This example is based on [!citet](Oldenburg1995OnTD), and [!cite](Elder1967TransientCI). The natural convection of water in a 2D porous tank heated on one part of the bottom is modelled. The schematic is demonstrated in [schematicFig]. The experiment details are elaborated in the referred paper so will not be shown here to avoid repetition. Users can refer to [This page](https://mooseframework.inl.gov/modules/porous_flow/governing_equations.html) for governing equations.
+
+
+!table id=datatable caption=Parameters of the problem [!citep](Oldenburg1995OnTD).
+| Symbol | Quantity | Value | Units |
+| --- | --- | --- | --- |
+| $\phi$ | Porosity | $0.1$ | $-$ |
+| $k$ | Permeability | $1.21*10^{-10}$  | $m^2$ |
+| $\mu$ | Viscosity ($T=20^oC$) | $1*10^{-3}$  | $J.Kg^{-1}.s^{-1}.m^{-1}$ |
+| $K$ | Thermal conductivity | $1.49$  | $J.Kg^{-1}.^oC^{-1}$ |
+| $C$ | Heat capacity of rock | $0$ | $m^2$ ||
+| $g$ | Gravity | $9.81$  | $m.s^{2}$ |
+| $T_0$ | Initial temperature | $12$  | $^oC$ |
+
+!media media/porous_flow/Convec_schematic.png style=width:100%;margin-left:10px; caption=The experiment schematic [!citep](Oldenburg1995OnTD). id=schematicFig
+
+
+## Input file setup
+
+This section describes the input file syntax.
+
+### Mesh
+
+The first step is to set up the mesh. In this problem, a rectangular tank with a width of 300 m and a height of 150 m is simulated. For convenience, instead of the YZ coordinate as shown in [schematicFig], the simulation was set up in XY coordinate with the positive Y direction pointing toward the top of the tank. In this code block, the numbers of nodes in the X and Y directions are specified along with the dimension of the tank. A notable feature is the declaration of the boundary "Heater". This is due to the fact that the heat element at the bottom of the tank just located between 0< x < 150 m. This enables simple boundary condition implementation at later stages.
```

Comment:
```suggestion
The first step is to set up the mesh. In this problem, a rectangular tank with a width of 300 m and a height of 150 m is simulated. For convenience, instead of the YZ coordinate as shown in [schematicFig], the simulation was set up in XY coordinate with the positive Y direction pointing toward the top of the tank. In this code block, the numbers of nodes in the X and Y directions are specified along with the dimension of the tank. A notable feature is the declaration of the boundary `heater`. This is due to the fact that the heat element at the bottom of the tank is located between 0< x < 150 m. This enables simple boundary condition implementation at later stages.
```